### ISSUE TITLE
Change creeps appearance based on their level

### ISSUE THREAD

**Issue by SanctusAnimus** [darklordabc/custom_hero_chaos_2#6]
Including (but not limited to):
* Model size/color/entire look
* Status and particle effects
* Some equipment parts (attachments BMD lib, may affect performance, needs deeper investigation)
* Special icons for especially dangerous creeps

Any other ideas?

**Comment by darklordabc** [comment]
I was mainly thinking skins.

**Comment by SanctusAnimus** [comment]
Yeah, but i mean, for example, we have same creeps on wave 10, 40, 70... 500+, how are we gonna spread skins on that? Or just set skin of most difficult after some wave?

**Comment by darklordabc** [comment]
Some of this will be determined by how it is too change skins, if its very hard, we might have fewer, if its trivial, we could have 10 per creep type.

**Comment by SanctusAnimus** [comment]
It's mainly based on what do you mean by **"skin"**, if it's about model and texture changed to another, than it's relatively simple (unless we talk about uploading new models, converting to Valve resource formats is quite a mess)

**Comment by darklordabc** [comment]
Texture change.
